pom.xml

```diff
@@ -66,6 +69,31 @@
 			<version>${vertx-unit.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.vertx</groupId>
+			<artifactId>vertx-kafka-client</artifactId>
+			<version>${vertx-kafka-client.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
```

Comment:
Indeed, a subsequent patch will change it to compile scope. I can make it compile scope from the start if you prefer.